docs/playground/README.md

```diff
@@ -0,0 +1,90 @@
+
+* **Wallet name:** a string that can be edited by the user;
+* **Wallet funds:** amount in ADA that can be edited by the user (this is the amount at genesis; i.e., start of the emulation); and
+* **Contract endpoints:** These are zero to ten labelled buttons. The labels in all wallets are the same and they are the function names of the contract endpoints defined in the current contract.
+
+The list of current wallet events is an unbound, possibly empty list of sequentially linked events. Each event includes the following components:
+
+* **Event name:** the name of the contract endpoint associated with this event;
+* **Wallet name:** the wallet that this event originated from;
+* **Close button:** clicking this button deletes the corresponding event from the list; and 
+* **Event arguments:** zero to ten key-value pairs. The keys are just string labels, but the values may be structured. The minimal requirement is that the values are multiline strings. Optionally, we render them as either as a string or a nested list of key-value pairs (depending on the type of the corresponding argument of the contract endpoint).
+
+When the user clicks a contract endpoint button in a wallet, a corresponding event is appended to the sequence of wallet events. The wallet name is set to the wallet whose button was clicked. The event arguments are initialised to default empty arguments, where appropriate.
+
+Optionally, the individual events in the sequence of wallet events can be reordered by dragging them around.
+
+The wallet pane is only active after the current contract has been compiled successfully. In fact, only in this case is the correct set of wallet endpoints known. Execution of the event sequence may fail when one or more arguments to one or more events is not a legitimate value of the corresponding type of contract endpoint function. In this case, each event with malformed argument(s) ought to be tinted red, while all malformed argument(s) are highlighted.
+
+### Mockchain pane
+
+The mockchain pane has three major parts.
+
+1. A visual representation of the mockchain in the form of the UTxO graph. The UTxO graph consists of nodes that represent individual (valid) transactions and edges that represent (valid) input-output connections.
+2. The final assignment of funds to wallets. This may be a subsets of the overall funds, because some funds may be locked by scripts. Optionally, the wallets should be colour coded and edges paying funds to those wallets should use the same colour. Moreover, we may display the wallet funds as coloured bars whose relative length represents their relative (final) value.
+3. A possibly empty list of *invalid* transactions, each of which indicates the reason for why it is invalid.
+
+Optionally, provide introspection facilities for valid and invalid transactions and graph edges. In particular, to query information that helps with debugging and also includes rejected, invalid transactions and failed script execution.
```

Comment:
More optionally, we could show the points when a blockchain trigger was activated for a wallet

Reply:
👍🏼